include/ck_tile/ops/gemm/kernel/grouped_gemm_kernel.hpp

```diff
@@ -131,10 +128,9 @@ struct GroupedGemmKernel
         return max(GemmPipeline::GetSmemSize(), EpiloguePipeline::GetSmemSize());
     }
 
-    CK_TILE_DEVICE void
-    Run(const GroupedGemmHostArgs& kargs, const index_t block_start, const index_t NBlock) const
+    CK_TILE_DEVICE void Run(const Hargs& kargs, const index_t block_start) const
     {
-        const auto [i_m, i_n] = TilePartitioner{}(block_start, NBlock);
+        const auto [i_m, i_n] = TilePartitioner{}(block_start, kargs.N);
```

Comment:
Just a comment for future. Later on when we would refactor all GEMM variations to reuse vanilla gemm kernel we would have to change this signature to sth like: `CK_TILE_DEVICE void Run(const Hargs& kargs) const`